packages/node/src/behavior/state/validation/conformance-compiler.ts

```diff
@@ -31,6 +32,9 @@ import {
 } from "./conformance-util.js";
 import { ValidationLocation } from "./location.js";
 
+/** A no-op function used to mark that a conformance expression is valid. */
```

Comment:
I think this is misleading, should be "used for valid enum values that have no specific conformance requirements"